### ISSUE TITLE
[Bug]: Publish to Environment stopped working

### ISSUE THREAD

**Issue by gntpet** [microsoft/AL-Go#1217]
### AL-Go version

AL-GO 5.3

### Describe the issue

Publish to Envirnoment suddenly fails with this error:

```
SRS_SRS.base_1.0.1944.0.app - Downloading AL Language Extension (Prerelease) to /home/runner/.bccontainerhelper/alLanguageExtension/15.0.1143290.zip
using Expand-Archive
/usr/bin/chmod: cannot access '/home/runner/.bccontainerhelper/alLanguageExtension/15.0.1143290/extension/bin/linux/altool': No such file or directory
ERROR: Command /home/runner/.bccontainerhelper/alLanguageExtension/15.0.1143290/extension/bin/linux/altool not found, you might need to install that command. [System.Management.Automation.RuntimeException]
```
my deployment settings are:

![image](https://github.com/user-attachments/assets/bc685437-b4fb-4d2d-ac9d-73acd189cfec)

### Expected behavior

"Publish to Environment" to work

### Steps to reproduce

Run "Publish to Environment"

### Additional context (logs, screenshots, etc.)

It did work on Wednesday

**Comment by freddydk** [comment]
Have the same issue reported from other partners - seems like the altool executable was removed - will investigate what happened.

**Comment by gntpet** [comment]
Thanks @freddydk , 
I also tried to deploy manually, but it does not work either. Give the following error message:

`Package deployment failed due to the following error(s): Failure status code ServiceUnavailable returned - Job Id : 89aeade1-9c38-4e3b-bad9-f6239ee796af`

Perhaps it's related issue

**Comment by freddydk** [comment]
> Thanks @freddydk , I also tried to deploy manually, but it does not work either. Give the following error message:
> 
> `Package deployment failed due to the following error(s): Failure status code ServiceUnavailable returned - Job Id : 89aeade1-9c38-4e3b-bad9-f6239ee796af`
> 
> Perhaps it's related issue

Manually? like in the BC Web Client? That is definitely not the same - this problem is caused by the altool not available in the allanguage.vsix package.

**Comment by freddydk** [comment]
Yes, I shipped a new BcContainerHelper which works with the faulty AL Language extension - should be working again
